python/tvm/relay/frontend/common.py

```diff
@@ -997,3 +1002,167 @@ def try_resolve_var_to_const(x, graph_params):
         return _op.const(value, dtype)
 
     return x
+
+
+class _SpanFiller(ExprMutator):
```

Comment:
we had been implementing a bunch of passes in C++, since the Python visitor infra involves a lot of calls that cross the FFI. I wonder if you'd be up for moving this into C++?

Reply:
I see. Currently we don't have plan to migrate them to a C++ pass. There are two reasons. 

1. First of all, considering the profiling result in the preRFC, the frontend conversion time is not increased too much. 
2. Second, the fill() and \_\_init\_\_() function heavily interact with different frontends. We would like to keep the flexibility to modify them via python script quickly. For example, in the [min_max_common ](https://github.com/apache/tvm/blob/b2058f4dd2e0ae1fc5ab51ac9f84b372a389a65a/python/tvm/relay/frontend/pytorch.py#L275)conversion of PyTorch, it might end up being a None, which is not common in other frontends. 

	
However, if the performance of frontend conversion is significantly decreased by SpanFiller in some cases, we can arrange a patch to move the mutator part (visitor funcions) to C++ pass. :)

Reply:
ok, given for now we want to invoke this after the import, we can keep it as Python for now